docs/backups_and_restore/dataimporters/pxc_dataimporter.md

```diff
@@ -0,0 +1,151 @@
+# DataImporter: Percona PXC operator
+
+The `everest-percona-pxc-operator` data importer enables you to import MySQL backups stored in external S3-compatible storage into clusters managed by Percona Everest.
```

Comment:
The `everest-percona-pxc-operator` data importer allows you to import backups taken using the Percona Operator for MySQL (XtraDb).